include/vctr/Containers/Span.h

```diff
@@ -31,14 +31,15 @@ namespace vctr
 template <class ElementType>
 consteval auto simdAlignedSpanStorageInfo() { return StaticStorageInfo<true, false, alignof (std::span<ElementType>)>(); }
 
-/** The view type.
+/** @ingroup Core
+    @brief The view type.
 
     Wraps a std::span and acts as a view to continuous data owned and stored somewhere else.
     All containers that offer a size and data interface are implicitly convertible into a Span.
     Note that unlike Vector and Array, Spans have a const ElementType in case they view non-mutable data. This means,
     e.g. a const Vector<Foo> will be converted into a Span<const Foo>.
  */
-template <class ElementType, size_t extent = std::dynamic_extent, class StorageInfoType = StorageInfo<std::span<ElementType, extent>>>
+template <class ElementType, size_t extent = std::dynamic_extent, class StorageInfoType = StorageInfo<std::span<ElementType, extent> > > // Extra whitespaces are needed for proper doxygen rendering
```

Comment:
That's weird. In that case you'll also need clang-format guards, won't you?

Reply:
see here: https://github.com/doxygen/doxygen/issues/9772

Good point about clang-format guards though